app/file.go

```diff
@@ -71,14 +72,27 @@ func ReadFile(path string) ([]byte, *model.AppError) {
 		accessKey := utils.Cfg.FileSettings.AmazonS3AccessKeyId
 		secretKey := utils.Cfg.FileSettings.AmazonS3SecretAccessKey
 		secure := *utils.Cfg.FileSettings.AmazonS3SSL
-		signV2 := *utils.Cfg.FileSettings.AmazonS3SignV2
+		encrypted := *utils.Cfg.FileSettings.AtRestEncryptKey != ""
+    signV2 := *utils.Cfg.FileSettings.AmazonS3SignV2
```

Comment:
oddly, when I run `make check-style` it doesn't report any issues, but when I run gofmt directly it does.  I'll update it.